data/resources/style-dark.css

```diff
@@ -33,3 +33,35 @@
 .sender-text-pink {
   color: #ff5694;
 }
+
+.sender-background-red {
+  background-color: #fb6169;
+}
+
+.sender-background-orange {
+  background-color: #faa357;
+}
+
+.sender-background-violet {
+  background-color: #b48bf2;
+}
+
+.sender-background-green {
+  background-color: #85de85;
+}
+
+.sender-background-cyan {
+  background-color: #62d4e3;
+}
+
+.sender-background-blue {
+  background-color: #65bdf3;
+}
+
+.sender-background-pink {
+  background-color: #ff5694;
+}
```

Comment:
I think it could be implemented more elegantly by using scss, but that's the task for the future

Reply:
What are they used for?

Reply:
Ah, for the separator color. Mh, I think we want a better to manage this. Let's keep the default style for now. We'll manage a better styling for later.